resource-managers/kubernetes/docker-minimal-bundle/src/main/docker/executor/Dockerfile

```diff
@@ -37,4 +37,6 @@ ENV SPARK_HOME /opt/spark
 WORKDIR /opt/spark
 
 # TODO support spark.executor.extraClassPath
-CMD exec ${JAVA_HOME}/bin/java -Dspark.executor.port=$SPARK_EXECUTOR_PORT -Xms$SPARK_EXECUTOR_MEMORY -Xmx$SPARK_EXECUTOR_MEMORY -cp ${SPARK_HOME}/jars/\* org.apache.spark.executor.CoarseGrainedExecutorBackend --driver-url $SPARK_DRIVER_URL --executor-id $SPARK_EXECUTOR_ID --cores $SPARK_EXECUTOR_CORES --app-id $SPARK_APPLICATION_ID --hostname $SPARK_EXECUTOR_POD_IP
+CMD SPARK_EXECUTOR_CLASSPATH="${SPARK_HOME}/jars/*" && \
+    if ! [ -z ${SPARK_EXECUTOR_EXTRA_CLASSPATH+x} ]; then SPARK_EXECUTOR_CLASSPATH="$SPARK_EXECUTOR_EXTRA_CLASSPATH:$SPARK_EXECUTOR_CLASSPATH"; fi && \
```

Comment:
We also do this in the driver docker file, to check the existence of an environment variable before using it.